src/scripts.js

```diff
@@ -1,168 +1,213 @@
   }
+  pantryArea.innerHTML = ''
+  pantry.forEach(ingInPantry => {
+    pantryArea.innerHTML += `<article class="single-ingredient-card">
+  <article class="ingredient-name-area">
+    <h2 class="ingredient-name">${ingInPantry.name}</h2>
+  </article>
+  <article class="ingredient-quantity-section">
+    <button>-</button>
+    <p class = 'ingredient-quantity'>${ingInPantry.amount}</p>
+    <button>+</button>
+  </article>
+</article>`
+  })
 }
 
+// function favoriteCard(event) {
+//   let specificRecipe = cookbook.recipes.find(recipe => {
+//     if (recipe.id  === Number(event.target.id)) {
+//       return recipe;
+//     }
+//   })
+//   if (!event.target.classList.contains('favorite-active')) {
+//     event.target.classList.add('favorite-active');
+//     favButton.innerHTML = 'View Favorites';
+//     user.addToFavorites(specificRecipe);
+//   } else if (event.target.classList.contains('favorite-active')) {
+//     event.target.classList.remove('favorite-active');
+//     user.removeFromFavorites(specificRecipe)
+//   }
+// }
 
-function displayDirections(event) {
-  let newRecipeInfo = cookbook.recipes.find(recipe => {
-    if (recipe.id === Number(event.target.id)) {
-      return recipe;
-    }
-  })
-  let recipeObject = new Recipe(newRecipeInfo, ingredientsData);
-  let cost = recipeObject.calculateCost()
-  let costInDollars = (cost / 100).toFixed(2)
-  cardArea.classList.add('all');
-  cardArea.innerHTML = `<h3>${recipeObject.name}</h3>
-  <p class='all-recipe-info'>
-  <strong>It will cost: </strong><span class='cost recipe-info'>
-  $${costInDollars}</span><br><br>
-  <strong>You will need: </strong><span class='ingredients recipe-info'></span>
-  <strong>Instructions: </strong><ol><span class='instructions recipe-info'>
-  </span></ol>
-  </p>`;
-  let ingredientsSpan = document.querySelector('.ingredients');
-  let instructionsSpan = document.querySelector('.instructions');
-  recipeObject.ingredients.forEach(ingredient => {
-    ingredientsSpan.insertAdjacentHTML('afterbegin', `<ul><li>
-    ${ingredient.quantity.amount.toFixed(2)} ${ingredient.quantity.unit}
-    ${ingredient.name}</li></ul>
-    `)
-  })
-  recipeObject.instructions.forEach(instruction => {
-    instructionsSpan.insertAdjacentHTML('beforebegin', `<li>
-    ${instruction.instruction}</li>
-    `)
-  })
-}
+// function cardButtonConditionals(event) {
+//   if (event.target.classList.contains('favorite')) {
+//     favoriteCard(event);
+//   } else if (event.target.classList.contains('card-picture')) {
+//     displayDirections(event);
+//   } else if (event.target.classList.contains('home')) {
+//     favButton.innerHTML = 'View Favorites';
+//     populateCards(cookbook.recipes);
+//   }
+// }
 
-function getFavorites() {
-  if (user.favoriteRecipes.length) {
-    user.favoriteRecipes.forEach(recipe => {
-      document.querySelector(`.favorite${recipe.id}`).classList.add('favorite-active')
-    })
-  } else return
-}
 
-function populateCards(recipes) {
```

Comment:
Thanks for removing this.  Pretty sure we are doing this already with one of our other functions in a cleaner way.  Cards are being populated with `displayAllRecipes()`